docs/static/releasenotes.asciidoc

```diff
@@ -76,12 +76,42 @@ This section summarizes the changes in the following releases:
 === Logstash 8.17.0 Release Notes
 
 [[known-issues-8-17-0]]
-==== Known issue
+==== Known issues
+
+[[known-issue-8-17-0-jvm]]
+===== JVM version changes needed when upgrading Logstash from 8.12.0 (or earlier)
+
+If the `jvm.options` file was modified and not overwritten with the newest version, you may see a "deserialize invocation error" message, causing the pipeline to crash. 
+Users are affected if the Persistent Queue (PQ) is enabled, and the pipeline is processing messages larger than 20MB.
+
+**Solution:** Apply the default change contained in the newer 'jvm.options' file, as seen in this https://github.com/elastic/logstash/blob/main/config/jvm.options#L66-L82[example].
+
+[[known-issue-8-17-0-json_lines]]
+===== "Input buffer full" error with {ls} 8.16.0, 8.16.1, or 8.17.0
+
+If you are using `json_lines` codec 3.2.0 (or earlier) with {ls} 8.16.0, 8.16.1, or 8.17.0, you may see an error similar to this one, crashing the pipelines:
```

Comment:
```suggestion
If you are using `json_lines` codec 3.2.0 (or later) with {ls} 8.16.0, 8.16.1, or 8.17.0, you may see an error similar to this one, crashing the pipelines:
```

Reply:
I wonder if we need to mention the version of the `json_lines` codec, as it is implied by the version of Logstash?

Reply:
(I'm trying to think of ways we can simplify the message)